comma/__init__.py

```diff
@@ -0,0 +1,64 @@
+import os
+import json
+import pandas as pd
+
+
+PARAMS_INDIVIDUAL = 'params_individual.json'
+PARAMS_MODEL = 'params_model.json'
+PARAMS_IPF_WEIGHTS = "ipf_weights.csv"
+
+
```

Comment:
I tend to keep the `__init__.py` file mostly empty; I think the functions here can be either deleted or moved to a more descriptive location.

Here, it might even make sense to add a `Hypothesis` class that implements all the functions for reading and validating those files.


> A commonly seen issue is to add too much code to __init__.py files. When the project complexity grows, there may be sub-packages and sub-sub-packages in a deep directory structure. In this case, importing a single item from a sub-sub-package will require executing all __init__.py files met while traversing the tree.

https://python-docs.readthedocs.io/en/latest/writing/structure.html#packages

Reply:
that's a great idea -- I'll make a to-do